cmd2/cmd2.py

```diff
@@ -3293,6 +3311,19 @@ def default(self, statement: Statement) -> bool | None:  # type: ignore[override
         self.perror(err_msg, style=None)
         return None
 
+    def completedefault(self, *_ignored: list[str]) -> list[str]:
+        """Call to complete an input line when no command-specific complete_*() method is available.
+
+        By default, it returns an empty list.
+
+        """
+        return []
+
+    def completenames(self, text: str, *_ignored: list[str]) -> list[str]:
```

Comment:
cmd2 does not use this method.

Reply:
Apparently both you and I added a few unit tests in the `test_completeion.py` file about 8 years ago which use `completenames` ;-). Who knew!

Reply:
@kmvanbrunt Here is one example of where we used the now otherwise unused `completenames` method in a unit test: https://github.com/python-cmd2/cmd2/blob/main/tests/test_completion.py#L227

What would be the most suitable replacement to use in these tests?